Tools/tsan/suppressions_free_threading.txt

```diff
@@ -11,34 +11,58 @@ race:set_allocator_unlocked
 
 ## Free-threaded suppressions
 
-race:_add_to_weak_set
-race:_in_weak_set
-race:_mi_heap_delayed_free_partial
-race:_PyEval_EvalFrameDefault
-race:_PyImport_AcquireLock
-race:_PyImport_ReleaseLock
-race:_PyInterpreterState_SetNotRunningMain
-race:_PyInterpreterState_IsRunningMain
-race:_PyObject_GC_IS_SHARED
-race:_PyObject_GC_SET_SHARED
-race:_PyObject_GC_TRACK
+race_top:_add_to_weak_set
+race_top:_in_weak_set
+race_top:_mi_heap_delayed_free_partial
+race_top:_PyEval_EvalFrameDefault
+race_top:_PyImport_AcquireLock
+race_top:_PyImport_ReleaseLock
+race_top:_PyInterpreterState_SetNotRunningMain
+race_top:_PyInterpreterState_IsRunningMain
+race_top:_PyObject_GC_IS_SHARED
+race_top:_PyObject_GC_SET_SHARED
+race_top:_PyObject_GC_TRACK
 # https://gist.github.com/mpage/0a24eb2dd458441ededb498e9b0e5de8
-race:_PyParkingLot_Park
-race:_PyType_HasFeature
-race:assign_version_tag
-race:gc_restore_tid
-race:initialize_new_array
-race:insertdict
-race:lookup_tp_dict
-race:mi_heap_visit_pages
-race:PyMember_GetOne
-race:PyMember_SetOne
-race:new_reference
-race:set_contains_key
-race:set_inheritable
-race:start_the_world
-race:tstate_set_detached
-race:unicode_hash
+race_top:_PyParkingLot_Park
+race_top:_PyType_HasFeature
+race_top:assign_version_tag
+race_top:gc_restore_tid
+race_top:initialize_new_array
+race_top:insertdict
+race_top:lookup_tp_dict
+race_top:mi_heap_visit_pages
+race_top:PyMember_GetOne
+race_top:PyMember_SetOne
+race_top:new_reference
+race_top:set_contains_key
+race_top:set_inheritable
+race_top:start_the_world
+race_top:tstate_set_detached
+race_top:unicode_hash
+race_top:Py_SET_TYPE
+race_top:_PyDict_CheckConsistency
+race_top:_PyImport_AcquireLock
+race_top:_Py_atomic_compare_exchange_int
+race_top:_Py_atomic_load_uintptr_relaxed
+race_top:_Py_dict_lookup_threadsafe
+race_top:__tsan_memset
```

Comment:
This one (`__tsan_memset`) seems weird. Can we get a function in CPython?

Reply:
Yeah. I'll also do that for `recv`.